src/pages/responsible-trading/_need-help.js

```diff
@@ -20,7 +22,8 @@ const help_content = [
         text2: (
             <Localize translate_text="GAMSTOP is a free service that enables you to self-exclude from all online gambling companies licensed in Great Britain." />
         ),
-        icon: <img src={GamStop} alt="gamstop" style={{ width: '91%', height: '100%' }} />,
+        icon: <img src={GamStop} alt="gamstop" style={{ width: '160px', height: '64px' }} />,
+        icon_mobile: <img src={GamStop} alt="gamstop" style={{ width: '113px', height: '34px' }} />,
```

Comment:
Should we declare and use `width` and `height` properties of image instead of in `style`

Reply:
Oh yes agreed. Good spot 👍